### DISCUSSION TITLE
`StyleDictionary.format['json/flat']()` equivalent in v4?

### DISCUSSION THREAD

**Discussion by kaelig** [amzn/style-dictionary#1282] [Q&A]
In v3 I'm able to call an existing format like so:

```js
StyleDictionary.format['json/flat']({
  dictionary,
  file,
  options,
  platform,
})
```

In v4, what's the equivalent?

**Comment by jorenbroekema** [comment]
https://github.com/amzn/style-dictionary/blob/main/lib/common/formats.js#L1398 it's still there but accessed like so:

```js
StyleDictionary.hooks.formats['json/flat']({
  dictionary,
  file,
  options,
  platform,
})
```

https://v4.styledictionary.com/version-4/migration/#formats
